Patches to GNU Classpath 0.07 so that it works on PPC64/AIX

@author Dave Grove

Copyright © IBM Corp. 2004

$Id$

Index: native/fdlibm/mprec.h
===================================================================
RCS file: /cvsroot/classpath/classpath/native/fdlibm/mprec.h,v
retrieving revision 1.1
diff -r1.1 mprec.h
296d295
< 
297a297
> #ifdef Pack_32
298a299,301
> #else
> #define MAX_BIGNUM_WDS 64
> #endif
